docs/UserGuide.md

```diff
@@ -45,125 +45,186 @@ AddressBook Level 3 (AB3) is a **desktop app for managing contacts, optimized fo
+* When editing tags, the existing tags of the project will be removed i.e. adding of tags is not cumulative.
+* You can remove all the project’s tags by typing `t/` without
     specifying any tags after it.
 
 Examples:
-*  `edit 1 p/91234567 e/johndoe@example.com` Edits the phone number and email address of the 1st person to be `91234567` and `johndoe@example.com` respectively.
-*  `edit 2 n/Betsy Crower t/` Edits the name of the 2nd person to be `Betsy Crower` and clears all existing tags.
+*  `project -e 1 n/ProjectX` Edits the project with `PROJECT_ID` 1 to be renamed to ProjectX.
+*  `project -e 2 r/AgentY/ProjectY` Edits the project with `PROJECT_ID` 2 to be associated with new repository link.
+*  `project -e 3 n/ProjectZ r/AgentZ/ProjectZ` Edits the project with `PROJECT_ID` 3.
 
-### Locating persons by name: `find`
+### Deleting a project : `project -d`
 
-Finds persons whose names contain any of the given keywords.
+Deletes a project from the application.
 
-Format: `find KEYWORD [MORE_KEYWORDS]`
+Format: `project -d PROJECT_ID`
 
-* The search is case-insensitive. e.g `hans` will match `Hans`
-* The order of the keywords does not matter. e.g. `Hans Bo` will match `Bo Hans`
-* Only the name is searched.
-* Only full words will be matched e.g. `Han` will not match `Hans`
-* Persons matching at least one keyword will be returned (i.e. `OR` search).
-  e.g. `Hans Bo` will return `Hans Gruber`, `Bo Yang`
+* Deletes the project with the specified `PROJECT_ID`.
+* The ID **must be a positive integer** 1, 2, 3, …​
 
 Examples:
-* `find John` returns `john` and `John Doe`
-* `find alex david` returns `Alex Yeoh`, `David Li`<br>
-  ![result for 'find alex david'](images/findAlexDavidResult.png)
+* `project -d 1` Deletes project with `PROJECT_ID` 1.
 
-### Deleting a person : `delete`
+### Tagging a client : `client -t`
 
-Deletes the specified person from the address book.
+Tags a client to a project.
 
-Format: `delete INDEX`
+Format: `client -t PROJECT_ID n/CLIENT_NAME [p/CLIENT_CONTACT_NUMBER] [e/CLIENT_EMAIL]…​`
 
-* Deletes the person at the specified `INDEX`.
-* The index refers to the index number shown in the displayed person list.
-* The index **must be a positive integer** 1, 2, 3, …​
+* Adds the client to the project with the specified `PROJECT_ID`.
 
 Examples:
-* `list` followed by `delete 2` deletes the 2nd person in the address book.
-* `find Betsy` followed by `delete 1` deletes the 1st person in the results of the `find` command.
+*  `client -t 1 n/Amy` Tags the project with `PROJECT_ID` 1 with a client with `CLIENT_NAME` Amy.
+*  `client -t 2 n/Bob p/91234567` Tags the project with `PROJECT_ID` 2 with a client with `CLIENT_NAME` Bob and 
+   `CLIENT_CONTACT_NUMBER` 91234567.
+*  `client -t 3 n/Charlie e/charlie@gmail.com` Tags the project with `PROJECT_ID` 2 with a client with `CLIENT_NAME` 
+    Charlie and `CLIENT_EMAIL` charlie@gmail.com.
+*  `client -t 2 n/Dave p/91111111 e/dave@gmail.com` 
```

Comment:
i think the explanation was missed out also 
```suggestion
*  `client -t 2 n/Dave p/91111111 e/dave@gmail.com` Tags the project with `PROJECT_ID` 2 with a client with `CLIENT_NAME` Dave, `CLIENT_CONTACT_NUMBER` 91111111 and `CLIENT_EMAIL` dave@gmail.com
```

Reply:
Made changes. Thank you!